### ISSUE TITLE
AST:ALT ratio high in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10456]
**Preferred term label:**
AST:ALT ratio high in blood

**Synonyms**
Aspartate aminotransferase to alanine aminotransferase ration increased greater than 1.5

**Definition (free text, please give PubMed ID)**
Aspartate aminotransferase to alanine aminotransferase ration increased greater than 1.5

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

- Hepatitis, acute alcoholic
- Alcoholic cardiomyopathy
- Alcoholic cirrhosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000813
- label: Elevated circulating aspartate aminotransferase to alanine aminotransferase ratio
- parents: [Abnormal circulating hepatic transaminase concentration (HP:0430135)](https://hpo.jax.org/app/browse/term/HP:0430135)
- pmid: 
- def: Aspartate aminotransferase to alanine aminotransferase ratio increased greater than 1.5.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10456
- synonyms:
